<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.exceptions.PropertyExceptionSpec" tests="3" skipped="0" failures="0" errors="0" timestamp="2017-05-03T08:30:56" hostname="testing-docker-6017ac84-a99f-499e-bd8d-5921f67c64fd" time="0.003">
  <properties/>
  <testcase name="Exception should have proper formatted message[0]" classname="griffon.exceptions.PropertyExceptionSpec" time="0.002"/>
  <testcase name="Exception should have proper formatted message[1]" classname="griffon.exceptions.PropertyExceptionSpec" time="0.0"/>
  <testcase name="Exception should have proper formatted message[2]" classname="griffon.exceptions.PropertyExceptionSpec" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
